patches/minecraft/net/minecraft/entity/EntityLiving.java.patch

```diff
@@ -1,14 +1,22 @@
 --- ../src-base/minecraft/net/minecraft/entity/EntityLiving.java
 +++ ../src-work/minecraft/net/minecraft/entity/EntityLiving.java
-@@ -171,6 +171,7 @@
+@@ -108,6 +108,7 @@
+ 
+     protected void func_184651_r()
```

Comment:
I recommend using this extension if using Chrome
https://chrome.google.com/webstore/detail/mcp-srg-mapper/ijjbogjfmngomjdmnabifgckbonmcebh